app/src/main/java/de/dreier/mytargets/base/db/migrations/Migration27.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2019 Florian Dreier
+ *
+ * This file is part of MyTargets.
+ *
+ * MyTargets is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2
+ * as published by the Free Software Foundation.
+ *
+ * MyTargets is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+package de.dreier.mytargets.base.db.migrations
+
+import android.content.ContentValues
+import android.database.sqlite.SQLiteDatabase
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import de.dreier.mytargets.shared.models.Dimension
+import de.dreier.mytargets.shared.models.Score
+import de.dreier.mytargets.shared.models.Target
+import de.dreier.mytargets.shared.models.db.End
+import de.dreier.mytargets.shared.models.db.Shot
+import org.threeten.bp.LocalTime
+import org.threeten.bp.format.DateTimeFormatter
+import timber.log.Timber
+
+object Migration27 : Migration(26, 27) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        Timber.i("Migrating DB from version 26 to 27")
+
+        database.execSQL(
+            "UPDATE RoundTemplate SET shotsPerEnd=3, endCount=10 " +
+                    "WHERE standardRoundId IN (SELECT id FROM StandardRound WHERE name ='Bray I')"
+        )
+        database.execSQL(
+            "UPDATE RoundTemplate SET shotsPerEnd=3, endCount=10 " +
+                    "WHERE standardRoundId IN (SELECT id FROM StandardRound WHERE name ='Bray II')"
+        )
+        database.execSQL(
+            "UPDATE Round SET shotsPerEnd=3, maxEndCount=10 " +
+                    "WHERE trainingId IN (SELECT t.id FROM Training as t JOIN StandardRound as s WHERE s.name ='Bray I')"
+        )
+        database.execSQL(
+            "UPDATE Round SET shotsPerEnd=3, maxEndCount=10 " +
+                    "WHERE trainingId IN (SELECT t.id FROM Training as t JOIN StandardRound as s WHERE s.name ='Bray II')"
+        )
+
+
+        database.query("SELECT `id`, `targetId`, `targetScoringStyleIndex`, `targetDiameter` FROM `Round` WHERE trainingId IN (SELECT t.id FROM Training as t JOIN StandardRound as s WHERE s.name in ('Bray I', 'Bray II'))")
```

Comment:
I'd suggest to break this query up into multiple lines so that it's easier to read.